client/src/components/tooltip/component.tsx

```diff
@@ -38,34 +47,59 @@ export const ToolTip: React.FC<TooltipProps> = ({
     ],
   });
 
+  const [isHovered, setIsHovered] = useState(false);
+
+  const arrowPositionClasses = {
+    top: '-translate-y-1/2 bottom-0',
+    right: '-translate-x-1/2 left-0',
+    bottom: 'translate-y-1/2 top-0',
+    left: 'translate-x-1/2 right-0',
+  };
+
   return (
     <Popover className="relative">
-      <Popover.Button>
-        <div ref={reference}>{children}</div>
-      </Popover.Button>
-      <Popover.Panel
-        ref={floating}
-        className={className}
-        style={{
-          position: strategy,
-          top: y ?? '',
-          left: x ?? '',
+      <Popover.Button
+        onMouseEnter={() => {
+          setIsHovered(true);
+        }}
+        onMouseLeave={() => {
```

Comment:
same, consider `useCallback`

Reply:
It won't probably matter, but I rather be consistent